examples/src/test/java/jdocs/jdbc/JdbcProjectionDocExample.java

```diff
@@ -56,11 +56,7 @@ public Order(String id, Instant time) {
   }
   // #repository
 
-  public OrderRepository orderRepository =
-      new OrderRepository() {
-        @Override
-        public void save(EntityManager entityManager, Order order) {}
-      };
+  public OrderRepository orderRepository = (entityManager, order) -> {};
```

Comment:
Compact but obscure. A real repository wouldn't have only one single method. I'd prefer to revert this.

Reply:
🆗  Reverted
I did that because this piece of code not shown in docs and nobody not seeing these except developers 😄

Reply:
we are also humans 🙂